### DISCUSSION TITLE
How to check for new firmware

### DISCUSSION THREAD

**Discussion by Thkos123** [Koenkk/zigbee2mqtt#19456] [General]
I have installed Zigbee2Mqtt and the frontend on a Pi. Frontend works fine, but I am struggling with the firmware updates.
I have added 1 device : a blitzwulf switch (Yuya). 
The OTA pages shows nothing, no device no current version, or new version.
I have read https://www.zigbee2mqtt.io/guide/usage/ota_updates.html

It should be able to manually check from the command line by sending a message. zigbee2mqtt/bridge/request/device/ota_update/check
My attempts failed I have no clue how to do that .

Any advice on how to check the firmware through the frontend or using the command line are welcome.
Thanks Theo

**Comment by Koenkk** [comment]
If the device doesn't show up under the OTA tab, OTA updates are not supported for this device.
